samples/guide/src/main/java/okhttp3/recipes/CustomTrust.java

```diff
@@ -39,21 +41,30 @@
 public final class CustomTrust {
   private final OkHttpClient client;
 
+  class KeyAndTrustManagers {
+    final KeyManager[] keyManagers;
+    final TrustManager[] trustManagers;
+
+    KeyAndTrustManagers(KeyManager[] keyManagers, TrustManager[] trustManagers) {
+      this.keyManagers = keyManagers;
+      this.trustManagers = trustManagers;
+    }
+  }
+
   public CustomTrust() {
-    X509TrustManager trustManager;
     SSLSocketFactory sslSocketFactory;
     try {
-      trustManager = trustManagerForCertificates(trustedCertificatesInputStream());
+      KeyAndTrustManagers keyAndTrustManagers = trustManagerForCertificates(trustedCertificatesInputStream());
       SSLContext sslContext = SSLContext.getInstance("TLS");
-      sslContext.init(null, new TrustManager[] { trustManager }, null);
+      sslContext.init(keyAndTrustManagers.keyManagers, keyAndTrustManagers.trustManagers, null);
```

Comment:
@swankjesse Should we change this test to use okhttp-tls?

Reply:
I did!

Reply:
Yes!

Reply:
For followup https://github.com/square/okhttp/issues/5701